packages/react/error-boundary/package.json

```diff
@@ -21,6 +21,7 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "@toss/react": "workspace:^1.4.0",
```

Comment:
`@toss/react`'s stable version is 1.5.0 now 
Is there a reason why you installed 1.4.0?

Reply:
Oh, I didn't check `@toss/react`'s stable version.
I will change this. Thanks 🙇‍♂️